dotcom-rendering/src/lib/decidePalette.ts

```diff
@@ -648,7 +648,7 @@ const textCardHeadline = (format: ArticleFormat): string => {
 	if (format.theme === ArticleSpecial.SpecialReportAlt)
 		return palette.specialReportAlt[100];
 
-	if (format.display === ArticleDisplay.Immersive) return BLACK;
+	if (format.display === ArticleDisplay.Immersive) return news[300];
```

Comment:
Given [using `news` directly is deprecated](https://github.com/guardian/csnx/blob/8a1e22c413d07c522dead9af23478264f9361acb/libs/%40guardian/source-foundations/src/index.ts#L42C1-L43) - should this be

```suggestion
	if (format.display === ArticleDisplay.Immersive) return palette.news[300];
```